security-proxy/security-proxy.properties

```diff
@@ -1,19 +1,18 @@
 # ------   proxy-servlet.xml   ---------
-public.host=https://georchestra.mydomain.org/
```

Comment:
`public.host=${publicUrl}` ?
To make surpercharging it more obvious

Reply:
Done with 97151e0